### DISCUSSION TITLE
How to delete the outer HTML tag using the keyboard shortcut dst without deleting the child tag

### DISCUSSION THREAD

**Discussion by dil-mnasir** [kylechui/nvim-surround#260] [Q&A]
I am trying to delete the outer HTML tag using the keyboard shortcut `dst`, but it is deleting the child tag instead. I have attached a video of the issue. Can someone help me fix this?

https://github.com/kylechui/nvim-surround/assets/93813665/6b52eece-9b6f-401c-921f-039b9f1f96d9

**Comment by kylechui** [comment]
Have you tried updating the plugin?

**Reply by dil-mnasir** [reply]
mini.ai was the culprit. After removing it worked.
